client/gutenberg/extensions/contact-form/editor.js

```diff
@@ -260,7 +260,7 @@ registerJetpackBlock( 'field-telephone', {
 
 registerJetpackBlock( 'field-textarea', {
 	...FieldDefaults,
-	title: __( 'Multi-line text' ),
+	title: __( 'Message' ),
 	keywords: [ __( 'Textarea' ), 'textarea', __( 'Message' ) ],
```

Comment:
If we go with this we should also update or remove the keyword ` __( 'Message' )` from the keywords array